examples/apps/cli/main.c

```diff
@@ -133,6 +134,9 @@ int main(int argc, char *argv[])
     {
         otTaskletsProcess(instance);
         otSysProcessDrivers(instance);
+#if OPENTHREAD_UART_RTT_ENABLE
+        utilsUartRttUpdate();
```

Comment:
The function `utilsUartRttUpdate()` is a platform defined function. It should be called by the function `otSysProcessDrivers()`， rather than calling it in the main.

Reply:
Its part of the utils. Since only RTT is needed there is no platform dependent code in this PR.

Reply:
It is part of the utils, but it also belongs to the `examples/platforms/`.  And the Simulation and Posix platform are unable to use the RTT Uart. So the RTT Uart is the platform code, and it only be available on certain platforms. I think the `main.c` is not a good place to call `utilsUartRttUpdate()`.

For the project ot-nrf528xx, the `utilsUartRttUpdate()` should be added to [ot-nrf528xx/src/src/system.c#L142](https://github.com/openthread/ot-nrf528xx/blob/main/src/src/system.c#L142).